### PR TITLE
source dist name

### PR DESCRIPTION
**Pull request by pjfanning**
see https://github.com/apache/incubator-pekko/pull/468

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -26,7 +26,7 @@ object Dependencies {
     // We don't force Pekko updates because downstream projects can upgrade
     // themselves. For more information see
     // https://pekko.apache.org//docs/pekko/current/project/downstream-upgrade-strategy.html
-    val pekko = "1.0.0-RC3"
+    val pekko = "1.0.0"
     val pekkoBinary = "current"
     val pekkoHttp = "0.0.0+4456-9a4c0fc7-SNAPSHOT"
```
Comment (He-Pin) [inline project/Dependencies.scala:31]: when will pekko http be updated to 1.0.0?

**Comment by He-Pin** [comment]
@pjfanning There is  a conflict need to be resolved

File: project/Dependencies.scala (hunk 1 above)
Comment (mdedetrich) [reply project/Dependencies.scala:31]: @pjfanning That's a fair point, should update pekko http to the latest snapshot that depends on pekko 1.0.0

**Comment by pjfanning** [comment]
apologies - rebased now

File: project/Dependencies.scala (hunk 1 above)
Comment (pjfanning) [reply project/Dependencies.scala]: rebase is done now

**Review by gmethvin — APPROVED** [review]

**Pull request merged**